src/commands/cmd_geo.cc

```diff
@@ -604,6 +624,8 @@ class CommandGeoSearchStore : public CommandGeoSearch {
     return Status::OK();
   }
 
+  static std::vector<CommandKeyRange> Range(const std::vector<std::string> &args) { return {{1, 1, 1}, {2, 2, 1}}; }
```

Comment:
Maybe we can just return `{1,2,1}` here?

Reply:
yes, this will work too, personally i want to separate them (more explicit?)
but i am ok with both of them, i can change it if you ask

Reply:
fine to me.